types/src/crypto/ear/keys.rs

```diff
@@ -78,10 +72,6 @@ pub type IdentityLinkKey = Key<IdentityLinkKeyType>;
 
 impl EarKey for IdentityLinkKey {}
 
-impl KdfDerivable<ConnectionKey, PseudonymousCredentialTbs, AEAD_KEY_SIZE> for IdentityLinkKey {
```

Comment:
`IdentityLinkKeyType` and the associated types and imples should all be unused now.

Reply:
Remove in d091c37